content/manuals/security/for-admins/hardened-desktop/settings-management/compliance-reporting.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Understand how to use the Desktop settings reporting dashboard
+keywords: Settings Management, docker desktop, hardened desktop, reporting, compliance
+title: Desktop settings reporting
+linkTitle: Desktop Settings Reporting
+weight: 30
+---
+
+Desktop settings reporting is a feature of Desktop Settings Management that tracks and reports user compliance with the settings that are assigned to them. This lets administrators to track the application of settings and monitor what actions they need to take to make users compliant.
```

Comment:
@ajthilakan is this the right way to phrase this?
> Desktop settings reporting is a feature of Desktop Settings Management

Reply:
+1 this is sufficient for current purposes. 

I should note that we have an upcoming roadmap item to introduce a "Minimum DD Version Policy" which prescribes a minimum DD version for their employees. This will also have a reporting element, which raises some UX questions (do we manage and report on policies holistically, or within these distinct policy features) that I dont believe we have answers to ye